builds/serverless-config-pack/serverless-java.yml

```diff
@@ -98,7 +98,7 @@ resourcesDisabled:
     #End:isSQSEventEnabled
     #Start:isSQSEventEnabled
     #Start:isSqsResourceNotExist
-    {event_source_sqs}:
+    SQSQueueName:
```

Comment:
nitpick - can we use `sourceQueue` instead of `SQSQueueName`?

Reply:
Done